src/parser/expression.js

```diff
@@ -385,6 +388,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("dynamicImport")) this.unexpected();
+
+      node = this.startNode();
+      this.next();
+      if (!this.match(tt.parenL)) {
+        this.unexpected();
```

Comment:
Maybe we can do a 
```
this.unexpected(null, `Unexpected token, expected (`);
```

https://github.com/babel/babylon/pull/150/files#diff-51a873a9697282bed0c15722a6ccf4e6R77

not sure if you can just use this.expect instead